todoapp/users/serializers.py

```
@@ -48,9 +47,19 @@ class Meta(UserSerializer.Meta):
 
 class UserRegisterSerializer(UserSerializer):
     password = serializers.CharField(write_only=True)
+    confirm_password = serializers.CharField(write_only=True)
 
     class Meta(UserSerializer.Meta):
-        fields = ["first_name", "last_name", "email", "date_joined", "password"]
+        fields = ["email", "first_name", "last_name", "password", "confirm_password"]
+
+    def validate(self, attrs):
```

Ruff (0.12.2)

Missing return type annotation for public function `validate`

(ANN201)